config.toml

```
@@ -222,7 +222,7 @@ language_alternatives = ["en"]
 
 [languages.es]
 title = "Kubernetes"
-description = "Production-Grade Container Orchestration"
+description = "Orquestación de contenedores para producción"
```

Well, I'm not sure if this a better suggestion
```suggestion
description = "Orquestación de contenedores para ambientes de producción"
```

---

I took the text from https://k8s.io/es/

I can propose changing both, so that they are synchronised. Should I do that?

---

Let me verify this with the rest of the team, maybe they decided to choose this because it's shorter.

---

We're still discussing the best approach, we will have something decided in a couple of days.

```suggestion
description = "Orquestación de contenedores para entornos productivos"
```

---

OK cool. I'm happy to commit whatever the ES localization team recommend!